mydoc/mydoc_cmor3_api.md

```diff
@@ -333,6 +333,63 @@ Returns upon success:
 {:.coral} 
 
 
+---
+
+### cmor_set_crs()
+
+Fortran: error_flag = cmor_set_crs(grid_id, mapping_name, parameter_names, parameter_values, parameter_units, text_parameter_names, text_parameter_values)
+{:.green}
+
+C: error_flag = cmor_set_crs(int grid_id, char *mapping_name, int nparameters, char **parameter_names, int lparameters, double parameter_values[], char **parameter_units, int lunits, int ntextparameters, char **text_parameter_names, int ltextparameters, char *text_parameter_values, int text_parameter_length[]) 
+{:.blue} 
+
+Python: set_crs(grid_id, mapping_name, parameter_names, parameter_values=None, parameter_units=None) 
+{:.coral} 
+
+
+*Description*: Define the Coordinate Reference System (CRS) variable that contains grid mapping parameters associated with a grid (see CF conventions for more info on which parameters to set). Similar to [cmor_set_grid_mapping](#cmor_set_grid_mapping), but allows for the setting of text-based parameters.
+
+*Note on `crs_wkt`*: This function allows for the setting of a [`crs_wkt`](https://cfconventions.org/cf-conventions/cf-conventions.html#use-of-the-crs-well-known-text-format){:target="_blank"} text attribute, meant to hold a Well-known Text (WKT) formatted string. However, CMOR will not validate the contents of this string.
+
+
+*Arguments*:
```

Comment:
@mauzey1 are any of these arguments optional - we assume certain entries, or are all required?

I have to stand up a test for the DRCDP data using this CRS, so will be happy to be pointed at a test example so I can attempt to create a simple, but complete example for our user - this is who I was hoping to get feedback from before we finalize the CMOR 3.11 release

Reply:
All of the text attributes, including `crs_wkt`, are optional.